nodestream/pipeline/value_providers/static_value_provider.py

```diff
@@ -24,6 +24,9 @@ def many_values(self, _: ProviderContext) -> Iterable[Any]:
     def is_static(self) -> bool:
         return True
 
+    def __str__(self):
```

Comment:
StaticProvider does not have any posibility of errors in single or many_values methods